_includes/url/url-base-path.html

```diff
@@ -0,0 +1 @@
+./
```

Comment:
What is this file?

Reply:
Abstracted URL so I could change it in once place. With this one in particular, I had to experiment a bit with how to encode URLs because `/` would point to `localhost:4000/` locally, but on `aaronsarnat.github.io/hierplane/` it would point to `aaronsarnat.github.io` instead of the folder where all the github pages stuff lived, and thus was breaking stuff. This made it easy to monkey around with that.

Reply:
Can you show me an example of where it's ued?

Reply:
Also, does jekyll not have some notion of variables?

Reply:
Couple of examples of where it's used:

https://github.com/aaronsarnat/hierplane/blob/5485329bb37582d7002e429070ab9939d4439b49/_includes/meta.html#L4

https://github.com/aaronsarnat/hierplane/blob/5485329bb37582d7002e429070ab9939d4439b49/_includes/scripts.html#L4

https://github.com/aaronsarnat/hierplane/blob/5485329bb37582d7002e429070ab9939d4439b49/_includes/url/url-getstarted.html

---

Jekyll does have variables: https://jekyllrb.com/docs/variables/

Would you prefer that I investigate experimenting with that instead?

Reply:
Yea, I mean, you've effectively implemented a variable (a pointer to a value, somewhere).

Hopefully jekyll's variables provide a more straightforward way to do this.  Maybe there's a `BASE_URL` attribute and all urls are made relative to that?

Reply:
Will take a stab at using Jekyll variables instead in next pass.

Reply:
Punted on addressing in this PR. Now tracking this issue via https://github.com/allenai/hierplane/issues/69